js/Common.js

```diff
@@ -628,6 +653,34 @@ if (!CRM.vars) CRM.vars = {};
               $el.select2('data', item, true);
             }
             return false;
+          })
+          // Pop-up Afform when clicking quick-add links
+          .on('click.crmEntity', '.crm-entityref-quick-add a', () => {
+            let url = $(this).attr('href');
```

Comment:
```suggestion
            let url = $('#select2-drop .crm-entityref-quick-add a').attr('href');
```

Reply:
For some reason, in my testing, `this` was referring to the wrong element. Changing this line fixes the quick-add behavior.

Reply:
... except when multiple quick-add buttons are configured, it only respects the first one. Hmmmm.

Reply:
@highfalutin Aha! I found "some reason" - it's because arrow functions [behave differently](https://stackoverflow.com/a/33308151/2918050) than regular functions with `this` binding. 
Fixed now, thanks for the hint.

Reply:
OK, this works for multiple quick-add buttons:

```suggestion
            let url = $(event.target).attr('href');
```